src/sync.rs

```diff
@@ -16,8 +16,8 @@ use actix_rt::System;
 use crossbeam_channel as cb_channel;
 use futures::channel::oneshot::Sender as SyncSender;
 use futures::{Future, StreamExt};
-use pin_project::pin_project;
 use log::warn;
+use pin_project::pin_project;
```

Comment:
Is pin_project being used here? I can't see it anywhere.

Reply:
My bad, I didn't noticed the removed line above. Still think the change should be dropped though, so we can keep git history clean.